### PR TITLE
CHMS-189 | Adding defaults for 'filters' parameter

### PR DESCRIPTION
**Pull request by abarriosr**

### CODE REVIEW

**Comment by abarriosr** [comment]
New method purge() has been added to the library.

File: src/ContentHub.php [hunk 1]
```diff
@@ -214,6 +214,23 @@ public function deleteEntity($uuid)
     }
 
     /**
+     * Purges all entities from the Content Hub.
+     *
+     * This method should be used carefully as it deletes all the entities for
+     * the current subscription from the Content Hub.
+     */
+    public function purge() {
+        $list = $this->listEntities();
```
Comment (scor) [inline src/ContentHub.php]: just a question. looks like we have 4 spaces here for indentation. Are we using a different coding standard for this PHP client library?

Comment (abarriosr) [inline src/ContentHub.php]: Yeap, this is not Drupal. We are using http://www.php-fig.org/psr/psr-2/

Comment (scor) [inline src/ContentHub.php]: ok that is fine. what tool do you use to follow this coding standard? do you have an IDE and a code style checker?

Comment (abarriosr) [inline src/ContentHub.php]: I used to have a special settings configured in PHP Storm but that blew up when my laptop died. I will have to set this up again.

**Pull request merged**